src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyleDiagnosticAnalyzerBase.cs

```diff
@@ -36,6 +38,9 @@ protected NamingStyleDiagnosticAnalyzerBase()
             SymbolKind.Property,
             SymbolKind.Parameter);
 
+        // HACK: RegisterSymbolAction doesn't work with locals & local functions
```

Comment:
:memo: This isn't a hack. It's a normal use of a stable, public API.

Reply:
fixed in 559efdf1f3247cf291db1097c031cbe1cc85a8c5